common/djangoapps/util/password_policy_validators.py

```diff
@@ -42,12 +44,12 @@ def validate_password_length(value):
     message = _("Invalid Length ({0})")
     code = "length"
 
-    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
-    max_length = getattr(settings, 'PASSWORD_MAX_LENGTH', None)
+    min_length = PASSWORD_MIN_LENGTH
```

Comment:
Well, to me, there are two solutions to this. One is to use function defined in https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/api.py#L599. The other is to define those settings. It's your call which solution you would like to apply.